server/utils/formatter.js

```diff
@@ -82,7 +82,10 @@ export function formatPayload(automation) {
  * @returns {Array} Formated automation to be returned by the API
  */
 export function formatAutomationResponse(payload) {
-  return payload.map(automation => formatPayload(automation));
+  if(!payload) {
+    return payload.map(automation => formatPayload(automation));
```

Comment:
I'm curious at this point. if payload is null/undefined/false, how does it execute `payload.map` in line 86, without throwing an error?

Reply:
Yeah. I think it's how the formatPayload method works. 

However, because my object doesn't return metadata, it doesn't fit into that method. Hence it throws undefined.

The workaround I found to still make use of that method is to add the if condition